_integration/testsuite/gatewayapi/001-path-condition-match.yaml

```diff
@@ -76,6 +76,7 @@ apiVersion: networking.x-k8s.io/v1alpha1
 kind: GatewayClass
 metadata:
   name: contour-class
+  namespace: projectcontour
```

Comment:
I thought that GatewayClass resources are still cluster-scoped? If that's the case then this will be discarded, I think?

Reply:
Yup it is and is ignored, but it's moving back to namespace scoped. I'll take out to match the v0.2.0 release we're tagged against.